src/dotnet/Users.Service/Db/DbRecent.cs

```diff
@@ -0,0 +1,41 @@
+using System.ComponentModel.DataAnnotations;
+using Stl.Versioning;
+
+namespace ActualChat.Users.Db;
+
+public class DbRecent : IHasId<string>, IHasVersion<long>, IRequirementTarget
+{
+    private DateTime _updatedAt;
+
+    public DbRecent() { }
+    public DbRecent(Recent recent) => UpdateFrom(recent);
+
+    [Key] public string Id { get; set; } = "";
+    public string ShardKey { get; set; } = null!;
+    public string Key { get; set; } = null!;
+    public string Scope { get; set; } = null!;
+    [ConcurrencyCheck] public long Version { get; set; }
+    public DateTime UpdatedAt {
```

Comment:
Maybe we should agree we never add this field if there is `Version`...

Reply:
this field is to order recent list, it's not informational only